cli/config_file.rs

```diff
@@ -293,7 +293,7 @@ impl ConfigFile {
   }
 
   pub fn new(text: &str, path: &Path) -> Result<Self, AnyError> {
-    let jsonc = jsonc_parser::parse_to_serde_value(text)?.unwrap();
+    let jsonc = jsonc_parser::parse_to_serde_value(text)?.unwrap_or(json!({}));
```

Comment:
This should error just like the import map would:
```suggestion
    let jsonc = match jsonc_parser::parse_to_serde_value(text) {
      Ok(Some(value)) if value.is_object() => value,
      _ => return Err(anyhow!("Unable to pass config file JSON")),
    };
```

Reply:
ok, I didn't know how import map was handled. It makes more sense.

Reply:
I'll apply the fix and update the test accordingly